src/OnchainKitProvider.tsx

```diff
@@ -30,11 +30,7 @@ export function OnchainKitProvider({
       chain: chain,
       rpcUrl: rpcUrl ?? null,
       schemaId: schemaId ?? null,
-      walletCapabilities: walletCapabilities ?? {
-        hasAtomicBatch: false,
-        hasAuxiliaryFunds: false,
-        hasPaymasterService: false,
-      },
+      walletCapabilities: walletCapabilities ?? {},
```

Comment:
This should not be store here anymore!

But instead our `useCapabilities` hook should memoize this info.

And long story short we should have a

```
import { useCapabilities } from `@coinbase/onchainkit`;
```

and just use that around the library to get the data.